src/BuildScriptGenerator/Helpers/StaticSiteGeneratorHelper.cs

```diff
@@ -32,6 +33,16 @@ public static bool IsStaticSite(ISourceRepo sourceRepo)
         /// <returns>True if the app is a Hugo app, false otherwise.</returns>
         public static bool IsHugoApp(ISourceRepo sourceRepo)
         {
+            // Check for Hugo environment variables
+            var environmentVariables = Environment.GetEnvironmentVariables();
```

Comment:
Make this method take in `IEnvironment` so that if we want to unit test, it can be easy